V14: Documenting your controllers (#6113)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/backoffice/document-your-own-api-in-swagger.md

﻿# Documenting your own API in swagger
When you open swagger, you are met by a list of 3 APIs:
![Default list of apis](images/DefaultListOfAPi.png)

> If we implement the api from the [Creating your own API](./create-your-own-api.md) article, it is originally under the `Management API` section. If we want to hove that, we can replace the `ManagementApiBaseController` attribute with a `Controller` attribute.

Reviewer: ```suggestion
If we implement the API from the [Creating your own API](./create-your-own-api.md) article, it is originally under the `Management API` section. If we want to move that, we can replace the `ManagementApiBaseController` attribute with a `Controller` attribute.
```

```csharp
[ApiController]
[VersionedApiBackOfficeRoute("my/item")]
[ApiExplorerSettings(GroupName = "My item API")]
public class MyItemApiController : Controller
```
When you open swagger, it will show up under the `Default API` section.
![Now it's under default API](images/DefaultListOfAPi.png)

> This is great if you only have 1 API, but if you have multiple, you might want to organise these as well as their own separate silos.

Reviewer: ```suggestion
It's great if you only have 1 API, however, if you have multiple, you might want to organize these as well as their own separate silos.
```
